types/plugins/ext_auth/config.proto

```diff
@@ -61,18 +81,18 @@ message AuthorizationRequest {
   // Sets a list of headers that will be included to the request to authorization service. Note that
   // client request of the same key will be overridden.
   repeated api.v1.HeaderValue headers_to_add = 1
-      [(validate.rules).repeated = {ignore_empty: true, min_items: 1}];
+  [(validate.rules).repeated = {ignore_empty: true, min_items: 1}];
```

Comment:
It seems you are using a different clang-format? You can install this one:
https://github.com/mosn/htnn/blob/0be21bc69650e3872672555ef76d843856a2ed98/tools/Dockerfile.dev#L70